java/fury-core/src/main/java/org/apache/fury/resolver/DisallowedList.java

```diff
@@ -52,6 +61,33 @@ class DisallowedList {
     }
   }
 
+  private static String calculateSHA256(Set<String> disallowedSet) {
```

Comment:
I think this would simpler if calculated the digest of the file bytes. That preserves the line endings so no need to add back separators.

Reply:
changed to use file. new contributors can use `sha256sum disallowed.txt` to get sum